src/constants/wizardForm.js

```diff
@@ -0,0 +1,62 @@
+import { Map } from 'immutable';
+
+const formListInterestsSportId = 'formListInterestsSportId';
```

Comment:
this id is the value to be used as translation ids, and it will be sent to the server and saved in db (not the translation). so, the value of inputs will be `formListInterestsSportId` or something like this.

Lets shorten it: 
const interestsSportId = 'interests.sport';